src/express-tooling.ts

```diff
@@ -82,18 +87,22 @@ function requireScopesMiddleware(scopes: string[],
  * @param options
  * @returns express middleware
  */
-function handleOAuthRequestMiddleware(options: MiddlewareOptions): RequestHandler {
+function handleOAuthRequestMiddleware(options: MiddlewareOptions,
+  logger?: Logger): RequestHandler {
 
   const {
     tokenInfoEndpoint,
     publicEndpoints
   } = options;
 
   if (!tokenInfoEndpoint) {
+    if (logger) {
+      logger.error('tokenInfoEndpoint must be defined');
```

Comment:
Do we catch this error? Because we throw already a TypeError we probably don't need to log it without additional details.

Reply:
Just thought it might have been a good idea to put it in the log as you have a more consistent view at one place when logging

Reply:
I agree with @ISO50 , if we throw an error already we do not have to log from my point of view, at least if there are no additional information.